app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title(page_title)
+    base_title = t "title"
+    page_title.empty? ? base_title : "#{page_title} | #{base_title}"
```

Comment:
Trả lời câu hỏi: Phân beiejt empty? và blank?

ở đây nên dùngblank? nhé em

Reply:
`empty?`: sử dụng trên các đối tượng dạng tập hợp(collection) như `Hash`, `Array`, `Set`, phương thức sẽ trả về `true` khi tập hợp không có phần tử nào. Có thể sử dụng `empty?` với `String` (bởi vì `String` là mảng các ký tự).
Khi sử dụng `empty?` thì ta phải biết chắc chắn kiểu của đối tượng đang sử dụng là `Array` hay `nil`. Nếu không biết trước điều này thì sử dụng nó sẽ không an toàn. Khi đó cần phải check điều kiện `nil?` trước.

`blank?`: `nil` và `false`chắc chắn là blank, `true` không phải là blank.
`Array` và `Hash` là blank khi chúng là empty. 
ta có `" ".blank? #=> true` khác với `String#empty?` chỉ kiểm tra trên độ dài của chuỗi thì `String#blank?` còn kiểm tra thêm cả khoảng trắng.